src/09-reveal/style.scss

```diff
@@ -0,0 +1,17 @@
+@charset "UTF-8";
+/*********************************************************************************************
+
+Name: Reveal
+Author:  Ministry of Justice
+
+**********************************************************************************************/
+
+/**
+* This file contains imported styling to support GOV UK Frontend usage in the plugin
+*
+* To import styles from govuk-frontend, simply include the Gov UK module using
+*  @imports with a URL reference
+**/
+
+// https://design-system.service.gov.uk/components/accordion/
+@import "node_modules/govuk-frontend/govuk/components/details/details";
```

Comment:
Just a heads up. Have you tested the accordion component with this integration? Also, the loading order of components matters so you may need to put this in the main style.scss file.

As far as I know styling in the accordion isn't compatible with some of the components in govuk-frontend so please beware.

Reply:
Seems to be working with accordion.

Reply:
Ok cool, we just need to make sure we test it each time we import a govuk component, and for sure if we import the button component, I'm quite sure this is what broke the accordion styling before.